### ISSUE TITLE
Brackets missing in HGVS Protein description

### ISSUE THREAD

**Issue by einicke** [varfish-org/varfish-server#1435]
**Describe the bug**
Brackets are gone for any protein position. As these are predictions they need to be there according to HGVS nomenclature

**To Reproduce**
1. Go to AnyCase
2. Look at any Variant
3. See error eg. p.I2067V

**Expected behavior**
There needs to be a bracket eg. p.(I2067V)
Carefull "p.?" is written without brackets, e.g. SpliceSiteVariants

**Additional context**
(https://hgvs-nomenclature.org/stable/background/simple/)](https://hgvs-nomenclature.org/stable/background/simple/#dna-rna-protein)

**Comment by holtgrewe** [comment]
I beg to differ.

It is understood that the protein impact is a prediction based on transcript level change as the assay is DNA and not protein based. Adding parentheses here adds no information but only adds complexity in parsing. Or, as Tufte says, "minimize non-data ink".

We could add a profile setting for HGVS that allows people to switch this on...

**Comment by holtgrewe** [comment]
Also, it's something to be changed in mehari as this is where the conversion comes from. We would need to add a parameter to add these parentheses when enabled.

**Comment by einicke** [comment]
I totally agree with you that adds no information at all. But I am not part of the HGVS Variant Nomenclature Committee ;)

From a clinical side, it creates a mess if it finds its way like this into reports.
If I export variants the column header is "Protein HGVS change" But it is not HGVS conform.

We should discuss what impact this has on automated exports

This can be moved to enhacement.

@xiamaz  Can you please update our export so that the brackets will appear in our database

**Comment by stolpeo** [comment]
Closed because moved to mehari. https://github.com/varfish-org/mehari/issues/417
